examples/scripts/compare_lithium_ion.py

```diff
@@ -16,7 +16,7 @@
 # create and run simulations
 sims = []
 for model in models:
-    sim = pybamm.Simulation(model)
+    sim = pybamm.Simulation(model, parameter_values=pybamm.ParameterValues("Chen2020"))
```

Comment:
Do we need to use Chen2020 for a specific reason or was this for developing reasons only? If the latter, I would vote for using default parameters.